src/components/NewMessageModal.vue

```diff
@@ -307,17 +307,26 @@ export default {
 				}
 
 				if (!this.composerData.id) {
-					const { id } = await saveDraft(data.account, dataForServer)
+					// This is a new message
+					const { id } = await saveDraft(dataForServer)
 					dataForServer.id = id
 					await this.$store.dispatch('outbox/enqueueMessage', {
 						message: dataForServer,
 					})
-				} else {
+				} else if (this.composerData.type === 0) {
```

Comment:
Magic number is a bit hacky but I'm fine for now as the next block is a generic else anyway.